scripts/bench.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
```

Comment:
bash may not be at `/bin/bash` — it's a good idea to write `#!/usr/bin/env bash` (the location of `env` is a bit more standard).